lib/pxutil.js

```diff
@@ -264,6 +266,29 @@ function getTokenObject(cookie, delimiter = ':') {
     return { key: '_px3', value: cookie };
 }
 
+function getGraphqlData(req) {
+    if (!req.path.includes('graphql')) {
+        return null;
+    }
+
+    const queryArray = req.body['query'] ? req.body['query'].split(/[^A-Za-z0-9_]/) : null;
+    if (!queryArray || queryArray.length < 1) {
+        return new GraphqlData(GraphqlOperationType.QUERY);
+    }
+
+    let operationType = queryArray[0];
+    let operationName = queryArray[1];
+    if (!Object.values(GraphqlOperationType).includes(operationType)) {
+        operationType = GraphqlOperationType.QUERY;
+        operationName = queryArray[0];
```

Comment:
I lost you here.
operationName = queryArray[0] ? In line 279 operationType = queryArray[0]

I can't follow the flow here. Try somehow to clear it out please

Reply:
Done